src/Cms/PageActions.php

```diff
@@ -726,6 +726,24 @@ public function duplicate(string $slug = null, array $options = [])
 		// create the slug for the duplicate
 		$slug = Str::slug($slug ?? $this->slug() . '-' . Str::slug(I18n::translate('page.duplicate.appendix')));
 
+		// if the item to be duplicated already exist
+		// add a suffix at the end of slug and title
+		$siblingKeys = $this->parentModel()->childrenAndDrafts()->keys();
+		if (in_array($slug, $siblingKeys) === true) {
+			$suffixCounter = 1;
```

Comment:
I am thinking to start with `2` instead. Then the sequence would be:

```
Some Page
Some Page Copy
Some Page Copy 2
```

I think that fits better than `Copy` to `Copy 1`.